src/proto/factories/layer_factory.cpp

```diff
@@ -239,6 +242,7 @@ Layer* construct_layer(lbann_comm* comm,
       return new slice_layer<layout, Dev>(comm,
                                           params.slice_axis(),
                                           slice_points);
```

Comment:
The scoping on this endif statement seems wrong. But I am look at it on an iPad at the moment.